extensions/ql-vscode/src/run-queries.ts

```diff
@@ -144,6 +160,11 @@ export class QueryEvaluationInfo {
     };
     try {
       await qs.sendRequest(messages.runQueries, params, token, progress);
+      if (qs.config.customLogDirectory) {
+        void showAndLogInformationMessage(
```

Comment:
Maybe this could be a warning instead, so it's harder to miss? (In case any users are relying on their logs being in a particular place)